docs/src/layout/page/demo/demoFiltersRail.jsx

```diff
@@ -77,6 +77,11 @@ function DemoFiltersRail(props) {
         <Page.FiltersRail
             isOpen={isOpen}
             isScrollable
+            filterOptions={{
+                isDirty: false,
+                onClear: () => {},
+                onApply: () => {},
```

Comment:
Would it be helpful to log something to console, so consumer of demo page knows these get called?
_**Not a Blocker**_ and discretionary.

Reply:
Fixed. [Commit Link](https://github.com/saddlebackdev/react-cm-ui/pull/474/commits/52151f36990e262214ff2e97224c94097b3c1aec)